cron/k8s/raw-transfer-v0.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 Security Scorecard Authors
```

Comment:
Use the file naming convention - `transfer-raw.yaml`. Remove `v0`.

Reply:
done, thanks